client/web/compose/src/views/Admin/Modules/Edit.vue

```diff
@@ -459,6 +458,7 @@ import RecordRevisionsSettings from 'corteza-webapp-compose/src/components/Admin
 import DataPrivacySettings from 'corteza-webapp-compose/src/components/Admin/Module/DataPrivacySettings'
 import ModuleTranslator from 'corteza-webapp-compose/src/components/Admin/Module/ModuleTranslator'
 import Validation from 'corteza-webapp-compose/src/components/Admin/Module/Validation'
```

Comment:
Since validation has been removed also remove its components and things that are not needed anymore